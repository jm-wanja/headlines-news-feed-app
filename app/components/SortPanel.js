import React from 'react';
import { RadioGroup, Radio } from 'react-mdl';

export default class SortPanel extends React.Component {
    render() {
        return (
            <div className='radioGroup' style={{ marginLeft: 300 }}>
                <RadioGroup name="sort" value="opt1">
                    <Radio className='top' value="opt1" ripple>Top</Radio>
                    <Radio className='latest' value="opt2" ripple>Latest</Radio>
                    <Radio className='popular' value="opt3" ripple>Popular</Radio>
                </RadioGroup>
            </div>
        );
    }
}