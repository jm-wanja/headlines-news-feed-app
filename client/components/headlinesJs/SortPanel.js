import React from 'react';
import { RadioGroup, Radio } from 'react-mdl';

export default class SortPanel extends React.Component {
    render() {
        return (
            <div>
                <RadioGroup name="sort" value="opt1">
                    <Radio value="opt1" ripple>Top</Radio>
                    <Radio value="opt2" ripple>Latest</Radio>
                    <Radio value="opt3" ripple>Popular</Radio>
                </RadioGroup>
            </div>
        );
    }
}