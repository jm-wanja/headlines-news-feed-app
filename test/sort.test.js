import React from 'react';
import { mount, shallow } from 'enzyme';
import shallowToJSON from 'enzyme-to-json';
import SortPanel from '../client/components/headlinesJs/SortPanel.js';

describe('sort panel component', () => {
    it('snapshot of SortPanel', () => {
        const sort = shallow(<SortPanel />);
        const tree = shallowToJSON(sort);
        expect(tree).toMatchSnapshot();
    })

    it('asserts there is a sort radiogroup', () => {
        const sort = shallow(<SortPanel />);
        expect(sort.find('#sort')).toBe.defined;
    })
})