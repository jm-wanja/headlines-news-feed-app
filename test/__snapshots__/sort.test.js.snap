// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sort panel component snapshot of SortPanel 1`] = `
<div>
  <RadioGroup
    container="div"
    name="sort"
    value="opt1"
  >
    <Radio
      ripple={true}
      value="opt1"
    >
      Top
    </Radio>
    <Radio
      ripple={true}
      value="opt2"
    >
      Latest
    </Radio>
    <Radio
      ripple={true}
      value="opt3"
    >
      Popular
    </Radio>
  </RadioGroup>
</div>
`;
